about.html

```diff
@@ -12,53 +12,122 @@
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
+  <div class="headernav">
+
+    <img class ="firstimg" src ="img/lambda-black.png">
+
```

Comment:
Since these anchor elements are identical, you do not need a separate class for each of them.